variables.tf

```
@@ -82,7 +83,7 @@ variable "iops" {
 
 variable "allocated_storage" {
   type        = number
-  description = "The allocated storage in GBs"
+  description = "(Required unless a `snapshot_identifier` or `replicate_source_db` is provided) The allocated storage in GBs"
```

please use this for the others too

```suggestion
  description = "The allocated storage in GBs. Required unless a `snapshot_identifier` or `replicate_source_db` is provided."
```

---

Done.